### PR TITLE
fix: input component

### PR DESCRIPTION
**Pull request by samb-local**
LOCAL-872

### CODE REVIEW

**Comment by davirds** [comment]
@samb-local it looks like the refresh is not updating the fiat input, only the `1 KUJI = ARS 3860.97` on the top of the card.

Screen Shot 2023-01-27 at 10 38 17 AM

**Comment by davirds** [comment]
@samb-local when I delete the value in crypto and click outside the field, it keeps the value in fiat and leaves the `open trade` button enabled. When I click the `open trade` button it shows me an error on the screen.

Screen Shot 2023-01-27 at 10 42 49 AM

**Comment by davirds** [comment]
@samb-local in the `My Offers`, when you try to edit an offer it show the value wrong.

before click edit:
Screen Shot 2023-01-27 at 10 57 41 AM
after click edit:
Screen Shot 2023-01-27 at 10 57 49 AM

**Review by davirds — CHANGES_REQUESTED** [review]
The code looks ok to me, I'm adding only some questions and improvements.

**Comment by samb-local** [comment]
> @samb-local it looks like the refresh is not updating the fiat input, only the `1 KUJI = ARS 3860.97` on the top of the card.
> 
> Screen Shot 2023-01-27 at 10 38 17 AM

fixed at https://github.com/Local-Money/localmoney/pull/153/commits/6e880c9eae7a177478c3568dfa7ff7eb50b7f9ea

**Comment by samb-local** [comment]
> @samb-local when I delete the value in crypto and click outside the field, it keeps the value in fiat and leaves the `open trade` button enabled. When I click the `open trade` button it shows me an error on the screen.
> 
> Screen Shot 2023-01-27 at 10 42 49 AM

fixed at https://github.com/Local-Money/localmoney/pull/153/commits/61b03d29498f27ea963a4ab4b81cfc7a0461279f

File: app/yarn.lock [hunk 1]
```diff
@@ -6565,7 +6565,7 @@ vite@^2.9.8:
 
 vue-currency-input@^2.0.0:
   version "2.5.1"
-  resolved "https://registry.yarnpkg.com/vue-currency-input/-/vue-currency-input-2.5.1.tgz#5349b94df8c9e99102a8960772748f477dfd5b95"
+  resolved "https://registry.npmjs.org/vue-currency-input/-/vue-currency-input-2.5.1.tgz"
```
[the remark this answers is not in the document]
Comment (samb-local) [reply app/yarn.lock]: fixed at https://github.com/Local-Money/localmoney/commit/6e880c9eae7a177478c3568dfa7ff7eb50b7f9ea

File: app/src/ui/components/CurrencyInput.vue [hunk 2]
```diff
@@ -1,42 +1,113 @@
...
-      const input = this.$refs.inputRef
-      input.focus()
-    },
-    update(val) {
-      this.setValue(val)
-    },
-  },
+<script setup>
+import { formatAmount } from '~/shared'
+const props = defineProps([
```
[the remark this answers is not in the document]
Comment (samb-local) [reply app/src/ui/components/CurrencyInput.vue]: fixed at https://github.com/Local-Money/localmoney/pull/153/commits/5c75ce82614695df130622f1f9a19f45001d6cd4

File: app/src/ui/components/CurrencyInput.vue [hunk 3]
```diff
@@ -1,42 +1,113 @@
...
-  methods: {
-    focus() {
-      const input = this.$refs.inputRef
-      input.focus()
-    },
-    update(val) {
-      this.setValue(val)
-    },
-  },
+<script setup>
```
[the remark this answers is not in the document]
Comment (samb-local) [reply app/src/ui/components/CurrencyInput.vue]: fixed at https://github.com/Local-Money/localmoney/pull/153/commits/5c75ce82614695df130622f1f9a19f45001d6cd4, it's `lang="ts"` actually. 👍🏽

**Comment by samb-local** [comment]
> @samb-local in the `My Offers`, when you try to edit an offer it show the value wrong.
> 
> before click edit: Screen Shot 2023-01-27 at 10 57 41 AM after click edit: Screen Shot 2023-01-27 at 10 57 49 AM

Fixed with other formatting issues at: https://github.com/Local-Money/localmoney/pull/153/commits/c71518ebc42fc750698be8776fae1c1e3a4d5f32

**Review by davirds — APPROVED** [review]
LGTM!

**Pull request merged**